climate_scanner/sentiment_classifier/sentiment_classifier.py

```diff
@@ -350,30 +356,22 @@ def rand_bin_array(K, N):
         text = text.iloc[0:100]
         return text
     
-    # just 
-    text = get_text()
+    # just to test
+    text = get_text()  # These 2 lines are going awya (just to test)
     txt_lst = text.Text.values.tolist()
 
-    def text_to_sentiment(txt_lst=txt_lst):
+    def text_to_sentiment(self,txt_lst=txt_lst):
```

Comment:
One thing to remember... where do I send the input data? Currently the input is in dictionary form as we discussed, not just a string.